core/controllers/contributor_dashboard.py

```diff
@@ -924,6 +927,74 @@ def get(
         self.render_json(self.values)
 
 
+class ContributorCertificateHandler(base.BaseHandler):
```

Comment:
`BaseHandler` needs to be parametrized

cc @sahiljoster32 Can we somehow enforce this?

Reply:
> cc @sahiljoster32 Can we somehow enforce this?

This is already enforced, without this MyPy checks will not pass. -- [LINK](https://github.com/oppia/oppia/actions/runs/3554331008/jobs/5970359568#step:7:126)